src/coreclr/jit/codegencommon.cpp

```diff
@@ -4224,7 +4227,7 @@ void CodeGen::genHomeSwiftStructParameters(bool handleStack)
         }
     }
 }
-#endif
+#endif // defined(SWIFT_SUPPORT) || defined(TARGET_RISCV64)
```

Comment:
Isn't this going to result in a lot of extra stack stores for RISCV?
At this point it seems like just factoring parts of this helper out and calling it from the same place as `genHomeRegisterParams` will end up being a simpler change overall.

Reply:
It didn't make unnecessary stores from what I examined the diffs but you're right, a RISC-V specific routine turned out much handier.